chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -209,14 +209,23 @@
 if node[:platform] == "suse"
   service "tftp" do
     # just enable, don't start (xinetd takes care of it)
-    enabled true
-    action [ :enable ]
+    enabled node[:provisioner][:enable_pxe] ? true : false
+    action node[:provisioner][:enable_pxe] ? "enable" : "disable"
+  end
+
+  # NOTE(toabctl): stop for tftp does not really help. the process gets started
+  # by xinetd and has a default timeout of 900 seconds which triggers when no
+  # new connections start in this period. So kill the process here
+  execute "kill in.tftpd process" do
+    command "pkill in.tftpd"
```

Comment:
I'm not a big fan of having a `pkill` every 15 minutes, but can't find a better alternative.